include/aikido/util/StepSequence.hpp

```diff
@@ -21,8 +19,11 @@ class StepSequence
   /// point. If includeEndpoints is true then the final point in the sequence
   /// will be the end point, even if it is at less than stepSize from the second
   /// to last point.
-  StepSequence(const double _stepSize, const bool _includeEndpoints = true,
-               const double _startPoint = 0.0, const double _endPoint = 1.0);
+  StepSequence(
+      const double _stepSize,
+      const bool _includeEndpoints = true,
+      const double _startPoint = 0.0,
+      const double _endPoint = 1.0);
```

Comment:
> Is this something we can enforce with clang-format?

I don't think so. `const` qualifier was added by us in a different PR. Let me remove it anyway.